NuGet.Config

```diff
@@ -10,5 +10,6 @@
     <add key="xunit" value="https://www.myget.org/F/xunit/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="AspNetCurrent" value="https://dotnet.myget.org/F/aspnet-feb2017-patch/api/v3/index.json" /> 
+    <add key="vstest" value="https://dotnet.myget.org/F/vstest/auth/967ad631-d24c-4c0e-bece-922de3064956/api/v3/index.json" />
```

Comment:
It appears this is a pre-authenticated feed URL.  What permissions does that access token give?  This token is now publicly available on the internet and anyone can use it.  Could they upload/delete packages on that feed with it?

Why not make the feed a public feed like all our other feeds?